_data/conferences.yml

```diff
@@ -289,13 +289,13 @@
 
 - title: CVPR
   hindex: 299
-  year: 2021
-  id: cvpr21
-  link: http://cvpr2021.thecvf.com/
+  year: 2022
+  id: cvpr22
+  link: http://cvpr2022.thecvf.com/
   deadline: '2020-11-16 23:59:59'
```

Comment:
Looks like you missed updating this :)

Reply:
Done.  This is the most important field!

It really is November 16 this year as it was last year:
http://cvpr2022.thecvf.com/submission-timeline